src/main/kotlin/com/exactpro/th2/kafka/client/Config.kt

```diff
@@ -28,28 +29,11 @@ import kotlin.time.DurationUnit
 import kotlin.time.toDuration
 
 class Config(
-    /**
-     * Matches th2 sessions with Kafka topics
-     */
-    val aliasToTopic: Map<String, KafkaTopic> = emptyMap(),
-
-    /**
-     * Matches th2 sessions with Kafka topics and keys
-     */
-    val aliasToTopicAndKey: Map<String, KafkaStream> = emptyMap(),
-
-    /**
-     * Matches th2 sessions with session groups
-     * Key: session group, value: list of session aliases
-     */
-    sessionGroups: Map<String, List<String>> = emptyMap(),
 
-    /**
-     * This session group will be set for outgoing messages whose
-     * session alias does not exist in the "sessionGroups" parameter.
-     */
-    defaultSessionGroup: String? = null,
+    @JsonUnwrapped
+    val defaultBookConfig: BookConfig,
```

Comment:
In this case we will not be able to specify more than one key from every topic: e.g. topicA+keyA and topicA+keyB because topic in this structure should be unique.